bind/plan.sh

```diff
@@ -33,10 +35,13 @@ do_prepare() {
     ln -sv "$(pkg_path_for file)/bin/file" /usr/bin/file
     _clean_file=true
   fi
+
+  pip install ply
```

Comment:
This is going to install into `/hab/pkgs/core/python2/<version>/<release>`, so will be unavailable at runtime.  Please update to ensure it gets installed into the bind package and the correct python ENV vars are set so that the scripts can use them.  

I'm not well versed in the python ecosystem, so I don't know what the right solution is here, but `core/curator` might be a good place to look.  I suspect that virtualenv + wrapping the script with an execution of `activate` to load the virtualenv is going to be required.  See `core/wget` for an example of wrapping a script.